requirements/base.txt

```diff
@@ -0,0 +1,3 @@
+dj-database-url==0.5.0
```

Comment:
This is a dependency of `django-environ`, so let's remove it.

Reply:
@webknjaz Now all good?